templates/cloud-init.sh

```diff
@@ -225,7 +231,7 @@ export KONG_PG_PASSWORD="$DB_PASSWORD"
 export KONG_PG_DATABASE="$DB_NAME"
 
 if [ "$EE_LICENSE" != "placeholder" ]; then
```

Comment:
Need to add a check whether an admin token has been specified to be used

```suggestion
if [ "$EE_LICENSE" != "placeholder" && ee_creds.admin_token != null ]; then
```

Reply:
In order to use the EE version, you need all three, so I think the earlier check is better. The problem with the suggestion here is that `EE_LICENSE` is a shell variable and `ee_creds.admin_token` is a Terraform one, so you can't check them both in the same conditional statement